src/components/MessagesList/MessagesGroup/Message/MessageButtonsBar/MessageForwarder.vue

```diff
@@ -134,17 +132,26 @@ export default {
 		},
 
 		openConversation() {
+			if (window.location.href.includes('/apps/files')) {
```

Comment:
Should have been:
```
if (!window.location.href.includes('/apps/spreed') && !window.location.href.includes('/call/')) {
```

Otherwise public share pages still don't work?